test/functional/connection.test.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
+const { withClient, setupDatabase } = require('./shared');
 const test = require('./shared').assert,
-  setupDatabase = require('./shared').setupDatabase,
   expect = require('chai').expect;
```

Comment:
```js
const { withClient, setupDatabase } = require('./shared');
const test = require('./shared').assert;
const { expect } = require('chai');
```
Couldn't make a suggestion for some reason but IYDM slowly but surely we will have consistent importing 😅